src/textual/app.py

```diff
@@ -1978,8 +1978,8 @@ async def invoke_ready_callback() -> None:
                                 with redirect_stdout(redirector):  # type: ignore
                                     await run_process_messages()
                         else:
-                            with redirect_stderr(None):
-                                with redirect_stdout(None):
+                            with open(os.devnull, "w") as thevoid:
```

Comment:
We prefer camel case words. I think there is a more apt term than "void" here. Suggest `null_file`.

Reply:
`null_file` makes the line too long and `black` complains. Should I reflow the line even though it's ugly, or should I use the "standard" `f` as a file handle?

Reply:
I'd break it out on to another line. Don't mind that. No single letter variables though...

Reply:
Ok, done.